packages/components/src/autocomplete/README.md

```diff
@@ -97,6 +97,14 @@ Whether to apply debouncing for the autocompleter. Set to true to enable debounc
 -   Type: `Boolean`
 -   Required: No
 
+#### position
+
+Use this string to modify the direction in which the popover should open relative to its parent node.
```

Comment:
What are other options available? It would be great to link to the list that covers them all.

Reply:
We can make the reference to the Popover component.